src/sparseml/pytorch/sparsification/quantization/modifier_quantization.py

```diff
@@ -494,50 +560,79 @@ def _enable_module_qat(self, module: Module):
                     )
                 )
             module_fuse_fn(**self._model_fuse_fn_kwargs)
-        elif self._model_fuse_fn_name is None:  # default auto fn
-            self._model_fuse_fn_kwargs["inplace"] = True
-            fuse_module_conv_bn_relus(module, **self._model_fuse_fn_kwargs)
-
-        activation_qconfig_kwargs = self._get_updated_activation_qconfig_kwargs()
 
-        # prepare each module / submodule for quantization
-        qconfig = get_qat_qconfig(
-            reduce_range=self._reduce_range,
-            activation_qconfig_kwargs=activation_qconfig_kwargs,
+        # build list of layer types that should not quantize output activations
+        to_remove_layer_name = []
+        if not self.quantize_linear_activations:
+            to_remove_layer_name.extend(LINEAR_ACTIVATION_NAMES)
+
+        if not self.quantize_conv_activations:
+            to_remove_layer_name.extend(CONV_ACTIVATION_NAMES)
+
+        if len(to_remove_layer_name) == 0:
+            to_remove_layer_name = None
+
+        # fix for freezing batchnorm statistics when not fusing BN with convs.
+        # pytorch only supports freezing batchnorm statistics for fused modules.
+        # this fix wraps BN modules adding with a new module class that supports
+        # methods related to freezing/unfreezing BN statistics.
+        configure_module_bn_wrappers(module)
+
+        # set qconfig.
+        # if tensorrt flag is used, set activation and weights to symmetric
+        # quantization.
+        # otherwise, use the default values set in QConfigProperties
+        qproperties = QConfigProperties(
+            activation_bits=self.activation_bits,
+            weight_bits=self.weight_bits,
+            activation_qconfig_kwargs=self.activation_qconfig_kwargs,
             weight_qconfig_kwargs=self.weight_qconfig_kwargs,
+            reduce_range=self.reduce_range,
         )
+        if self.tensorrt:
```

Comment:
let's add a log here stating that we are overriding quantization to target tensorrt (symmetric int8 quantization)

Reply:
Done.